gtest/CMakeLists.txt

```diff
@@ -14,8 +16,8 @@ if ( NOT GTest_FOUND )
     ExternalProject_Add(GTest
```

Comment:
Does the entire `ExternalProject_Add` command need to be duplicated between the Linux and MSVC if statement? It looks like almost everything is the same except the path and extensions of the libraries.

```cmake
# Common configuration
ExternalProject_Add(GTest
...
)
file(...)
set(GTEST_INCLUDE_DIRS ...)

# Compiler specific configuration
if(GNU / CLANG ...)
  set(GTEST_LIBRARIES ...)
  set(GTEST_MAIN_LIBRARIES ...)
elseif(MSVC)
  set(GTEST_LIBRARIES ...)
  set(GTEST_MAIN_LIBRARIES ...)
endif()
```

Reply:
updated